homework.md

```diff
@@ -39,6 +39,7 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 }
 ```
+no only the ones the the id h1
```

Comment:
Well done @ikanengKhoza
Can you have a look at this answer again?

Reply:
thanks